Project.toml

```diff
@@ -7,23 +7,27 @@ version = "0.1.8"
 ACSets = "227ef7b5-1206-438b-ac65-934d6da304b8"
 Catlab = "134e5e36-593f-5add-ad60-77f754baafbe"
 Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
-DelayDiffEq = "bcd4f6db-9728-5f36-b5f7-82caef46ccdb"
-DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 LabelledArrays = "2ee39098-c373-598a-b85f-a56591580800"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
-OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
```

Comment:
```suggestion
```

Is `Requires` needed here? This shouldn't be a dependency anymore since we are requiring Julia 1.9. It should also be removed from the `[compat]` section